src/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -10,9 +10,18 @@ import React, {
 import { Link, RouteComponentProps, withRouter } from 'react-router-dom';
```

Comment:
TxReceipt does not match design
https://www.figma.com/file/BY0SWc75teEUZzws8JdgLMpy/%5BMyCrypto%5D-Web%2FApp-Final-Designs?node-id=15945%3A0